ui/src/app/components/summary/summary.component.html

```diff
@@ -118,29 +118,8 @@
         </svg>
       </div>
       <div class="no-issue-message">
-        Woohoo! No issues or suggestions<br />found for {{ currentObject.name }}.
+        Woohoo! No issues or suggestions<br />found.
       </div>
     </div>
   </div>
-  <div class="no-object-container" *ngIf="!currentObject">
```

Comment:
Why is this removed ? Will this never be triggered ?

Reply:
this was been shown when no table was selected, after this code change if no table is selected then will show all the issues